src/pages/EntityDetails/Model/Logs/ActionLogs/_action-logs.scss

```diff
@@ -28,6 +28,13 @@
     }
   }
 
+  .entity-details__payload-modal {
+    position: relative;
+
+    /* Pulled from Vanilla's Modal backdrop */
```

Comment:
It would be helpful to add a permalink to where this is set in Vanilla, in case this breaks and we need to check if the Vanilla value has changed.